scripts/linters/pylint_extensions.py

```diff
@@ -1484,31 +1484,63 @@ class RestrictedImportChecker(checkers.BaseChecker):
             'domain layer and controller layer respectively.'),
     }
 
-    def visit_import(self, node):
-        """Visits every import statement in the file.
+    options = (
+        (
+            'forbidden-imports',
+            {
+                'default': ('core.storage:core.domain',),
+                'type': 'csv',
+                'metavar': '<comma separated list>',
+                'help': 'List of disallowed imports.'
+            }
+        ),
+    )
+
+    def iterate_forbidden_imports(self, node):
+        """Yields pairs of module name and forbidden imports.
 
         Args:
             node: astroid.node_classes.Import. Node for a import statement
                 in the AST.
+
+        Yields:
+            tuple(str, str). Yields pair of module name and forbidden import.
         """
+        splited_module_to_forbidden_imports = [
+            forbidden_import.strip().split(':')
+            for forbidden_import in self.config.forbidden_imports
+        ]
```

Comment:
Parse and initialize the `forbidden_imports` in the open function. ([ref](https://github.com/oppia/oppia/pull/11557#discussion_r557632214))

Reply:
Done.